install/kubernetes/cilium/templates/cilium-agent/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.agent (not .Values.preflight.enabled) }}
```

Comment:
I think we need to feature gate this via a Helm value. Otherwise this will be installed by default, which I don't think is what we want for an alpha K8s feature.

Reply:
This is a good idea. I'll add a same validation